docker-compose.dev.yaml

```diff
@@ -127,9 +125,7 @@ services:
       - "${BENTOV2_WES_DEBUGGER_EXTERNAL_PORT}:${BENTOV2_WES_DEBUGGER_INTERNAL_PORT}"
 
   drs:
-    build:
-      context: $PWD/lib/drs
-      dockerfile: Dockerfile.dev
+    image: ${BENTOV2_DRS_IMAGE}:latest-dev
```

Comment:
none of the other services have the `:latest-dev` tag explicitly here; what's the plan for consistency / development?

Reply:
no plan yet - this was meant as a workaround initially, but we can continue with it if it works!